src/springfield/fields.py

```diff
@@ -339,6 +340,20 @@ class UrlField(StringField):
     """
     :class:`Field` with a URL value
     """
+    def adapt(self, value):
+        """
+        Validate that the `value` has a valid URL format containing
+        a scheme and network location using urlparse.
+
+        Supported schemes include `http`, `https` and `ftp`.
```

Comment:
I think just requiring a scheme is enough. Until we implement more thorough validation we don't want to be too restrictive.